lib/octokit/connection.rb

```diff
@@ -206,5 +206,14 @@ def parse_query_and_convenience_headers(options)
 
       opts
     end
+
+    def response_data_correctly_encoded(response)
+      # TODO: I'm guessing base64 content responses are still broken, but this fixes the bug we're running into
```

Comment:
Would you mind removing this TODO comment and filing an issue instead?

Reply:
Done 🙂 c9aa273